app/views/admin/offers/index.html.erb

```diff
@@ -1,5 +1,4 @@
 <div class='page-title'>
-  <%= link_to 'Create New', new_admin_offer_path, class: 'btn btn-small btn-primary pull-right' %>
```

Comment:
Question: I'm unclear if this was intentional - can you confirm removing this link was meant to be a part of this PR? 🤔

Reply:
Yes, I realized while working on this that we don't create offers this way in the controller. It's done from a submission page.

Reply:
(It was copied over from another form incorrectly).